.github/workflows/CI.yml

```diff
@@ -6,6 +6,40 @@ on:
       - master
   pull_request:
 
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        group:
+          - Interface
+          - Integrators
+          - Regression
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: 1
+      - uses: actions/cache@v1
+        env:
+          cache-name: cache-artifacts
+        with:
+          path: ~/.julia/artifacts
+          key: ${{ runner.os }}-test-${{ env.cache-name }}-${{ hashFiles('**/Project.toml') }}
+          restore-keys: |
+            ${{ runner.os }}-test-${{ env.cache-name }}-
+            ${{ runner.os }}-test-
+            ${{ runner.os }}-
+      - uses: julia-actions/julia-buildpkg@v1
+      - uses: julia-actions/julia-runtest@v1
+        env:
+          GROUP: ${{ matrix.group }}
+      - uses: julia-actions/julia-processcoverage@v1
+      - uses: codecov/codecov-action@v1
+        with:
+          file: lcov.info
+
+
```

Comment:
There already exists a job of name `test`. It should be removed or extended.

Reply:
I don't know what you mean.

Reply:
The workflow contains two jobs of the same name, one for linux and one for Windows. They should be merged.

Reply:
That's also the reason for why the workflow is no valid and not even the tests on Github run.